doc/install.rst

```diff
@@ -127,8 +128,8 @@ and NumPy and SciPy are not recompiled from source, which can happen when using
 particular configurations of operating system and hardware (such as Linux on
 a Raspberry Pi).
 
-If you must install scikit-learn and its dependencies with pip, you can install
-it as ``scikit-learn[alldeps]``.
+If you must install scikit-learn and all its optional dependencies with pip, you
+can install it as ``scikit-learn[alldeps]``.
```

Comment:
I do not think this is needed anymore. This use to mean scipy and numpy: https://github.com/scikit-learn/scikit-learn/pull/6990 but was removed.

Reply:
Indeed, I did not understand that it meant numpy and scipy. Let's delete this paragraph as it's no longer correct indeed.